run.sh

```diff
@@ -21,23 +21,19 @@ elif ! system_has node; then
   echo "- https://github.com/creationix/nvm#usage"
   exit 1
 elif [ "$(printf '%s\n' "$requirednodeversion" "$currentnodeversion" | sort -V | head -n1)" != "$requirednodeversion" ]; then 
-     echo "The node version must be >= v6.2.0"
-     exit 1
-elif ! system_has yarn; then
-  echo "Yarn is mandatory to continue"
-  echo "Check this guide to complete the installation: https://yarnpkg.com/lang/en/docs/install/#alternatives-tab"
+  echo "The node version must be >= v6.2.0"  
   exit 1
 fi
 
-
-yarn global add create-react-app --prefix /usr/local > /dev/null 2>&1
+npm i -g create-react-app  > /dev/null 2>&1
 PROJECT_NAME=`echo "${PWD##*/}"`
 cd ..
-create-react-app "${PROJECT_NAME}"
-yarn global add yo generator-react-bootstrap > /dev/null 2>&1
+create-react-app "${PROJECT_NAME}" --use-npm
+npm i -g yo generator-react-bootstrap > /dev/null 2>&1
 cd $PROJECT_NAME
+echo "Project created"
 yo react-bootstrap --force
-yarn global add package-json-merge > /dev/null 2>&1
+npm i -g package-json-merge > /dev/null 2>&1
 package-json-merge package.json packageB.json > packageA.json
```

Comment:
This line is creepy, but is the best we can do at moment I think

Reply:
Merging both packages?... Yes I do agree, however we can create an array of minimum dependencies and do a yarn add of all of them below